presto-spark-launcher/src/main/java/com/facebook/presto/spark/launcher/PrestoSparkClientOptions.java

```diff
@@ -17,9 +17,12 @@
 
 public class PrestoSparkClientOptions
 {
-    @Option(name = {"-f", "--file"}, title = "file", description = "sql file to execute", required = true)
+    @Option(name = {"-f", "--file"}, title = "file", description = "local sql file to execute")
     public String file;
 
+    @Option(name = {"-ef", "--externalFile"}, title = "externalFile", description = "external sql file location to execute")
```

Comment:
Let's leave only the `--file`. The `PrestoSparkClientOptions` is only used in OSS, and it is configured to always read the local file system.